<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>BeliefPropQtGUIClass</class>
  <widget class="QMainWindow" name="BeliefPropQtGUIClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1071</width>
        <height>730</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>BeliefPropQtGUI</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <widget class="QLabel" name="targetImage">
        <property name="enabled">
          <bool>true</bool>
        </property>
        <property name="geometry">
          <rect>
            <x>30</x>
            <y>70</y>
            <width>501</width>
            <height>381</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="scaledContents">
          <bool>true</bool>
        </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
          <rect>
            <x>30</x>
            <y>470</y>
            <width>291</width>
            <height>41</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Run Stereo Processing</string>
        </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
          <rect>
            <x>30</x>
            <y>585</y>
            <width>141</width>
            <height>20</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Naive CPU</string>
        </property>
        <property name="checked">
          <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
        </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
          <rect>
            <x>130</x>
            <y>585</y>
            <width>121</width>
            <height>20</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Optimized CPU</string>
        </property>
        <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
        </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
          <rect>
            <x>260</x>
            <y>585</y>
            <width>95</width>
            <height>20</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>CUDA</string>
        </property>
        <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
        </attribute>
      </widget>
      <widget class="QLabel" name="label">
        <property name="geometry">
          <rect>
            <x>30</x>
            <y>540</y>
            <width>241</width>
            <height>31</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Implementation:</string>
        </property>
        <property name="textFormat">
          <enum>Qt::PlainText</enum>
        </property>
      </widget>
      <widget class="QLabel" name="label_2">
        <property name="geometry">
          <rect>
            <x>30</x>
            <y>20</y>
            <width>971</width>
            <height>41</height>
          </rect>
        </property>
        <property name="font">
          <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
          </font>
        </property>
        <property name="text">
          <string>Reference Image of Stereo Set</string>
        </property>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1071</width>
          <height>26</height>
        </rect>
      </property>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <connections/>
  <buttongroups>
    <buttongroup name="buttonGroup"/>
  </buttongroups>
</ui>
